src/elements/element.rectangle.js

```diff
@@ -50,42 +50,76 @@ module.exports = function(Chart) {
 	}
 
 	Chart.elements.Rectangle = Chart.Element.extend({
-		draw: function() {
+		draw: function(isHorizontalBar) {
+
 			var ctx = this._chart.ctx;
 			var vm = this._view;
-
-			var halfWidth = vm.width / 2,
-				leftX = vm.x - halfWidth,
-				rightX = vm.x + halfWidth,
-				top = vm.base - (vm.base - vm.y),
-				halfStroke = vm.borderWidth / 2;
+			var x1,	x2,	y1,	y2;
+			var sign = {};
+			var borderWidth = vm.borderWidth;
+			var borderSkipped = (vm.borderSkipped !== undefined)? vm.borderSkipped: 'bottom';
+
+			if ((isHorizontalBar === undefined) || (!isHorizontalBar)) {
```

Comment:
I think this could be simplified as `if (!isHorizontalBar)` since `undefined` will be converted to false